CHANGELOG.md

```diff
@@ -1,6 +1,55 @@
 # Changelog
 
-## v2.3.0
+## v2.4.0 (Unreleased)
+
+### Web Terminal In Argo CD UI
+
+Feature enables engineers to start a shell in the running application container without leaving the web interface. Just find the required Kubernetes
+Pod using the Application Details page, click on it and select the Terminal tab. The shell starts automatically and enables you to execute the required
+commands, and helps to troubleshoot the application state.
+
+### Access Control For Pod Logs & Web Terminal
+
+Argo CD is used to manage the critical infrastructure of multiple organizations, which makes security the top priority of the project. We've listened to
+your feedback and introduced additional access control settings that control access to Kubernetes Pod logs and the new Web Terminal feature.
+
+### OpenTelemetry Tracing Integration
+
+The new feature allows emitting richer telemetry data that might make identifying performance bottlenecks easier. The new feature is available for argocd-server
+and argocd-repo-server components and can be enabled using the --otlp-address flag.
+
+### Power PC and IBM Z Support
```

Comment:
Is this true for just the images, and not the CLI yet? https://github.com/argoproj/argo-cd/pull/8991